Purchases/Public/Purchases.swift

```diff
@@ -964,17 +966,17 @@ public extension Purchases {
      * - Parameter productIdentifiers: Array of product identifiers for which you want to compute eligibility
      * - Parameter receiveEligibility: A block that receives a dictionary of product_id -> ``IntroEligibility``.
      */
-    @objc
+    @objc(checkTrialOrIntroductoryPriceEligibility:completionBlock:)
     // swiftlint:disable line_length
     func checkTrialOrIntroductoryPriceEligibility(_ productIdentifiers: [String],
-                                                  completionBlock receiveEligibility: @escaping ReceiveIntroEligibilityBlock) {
+                                                  completion receiveEligibility: @escaping ReceiveIntroEligibilityBlock) {
```

Comment:
This completion block's naming doesn't match the others where we do:

```
  _ maybeCompletion: @escaping ReceiveIntroEligibilityBlock) {
```

Reply:
I'm actually going a different direction with this one, shortening it to just `completion`